compass/ocean/tests/mitgcm_baroclinic_gyre/__init__.py

```diff
@@ -0,0 +1,24 @@
+from compass.ocean.tests.mitgcm_baroclinic_gyre.gyre_test_case import (
+    GyreTestCase,
+)
+from compass.testgroup import TestGroup
+
+
+class MitgcmBaroclinicGyre(TestGroup):
+    """
+    A test group for MITgcm baroclinic gyre test cases
+    """
+    def __init__(self, mpas_core):
+        """
+        mpas_core : compass.MpasCore
+            the MPAS core that this test group belongs to
+        """
+        super().__init__(mpas_core=mpas_core, name='mitgcm_baroclinic_gyre')
+
+        for resolution in ['80km']:
```

Comment:
Consider adding a comment that resolutions here must be added with the 'km' suffix (since later you chop off the last 2 characters to get the float)

Reply:
An alternative would be to provide the resolution as a float and add the `km` suffix to the subdirectory.  I think that might be cleaner.  (I know we were following ziso, so that one should probably be fixed in a similar way.)

Reply:
By the way, I have something along these lines here, where the resolution is always given in meters
https://github.com/MPAS-Dev/compass/blob/f8af494f5c277c381d1b02320c1b9e297d01f9da/compass/ocean/tests/turbulence_closure/default/__init__.py#L38-L41

Reply:
@cbegeman Working on this `resolution` change now. A few questions/suggestions:
1) Are you satisfied with resolution in meters by default? It makes sense for turbulence closure since we go to fine resolution. Is it best to be consistent across cases? Happy to switch mine, just wanted to check. 
2) the comment blocks in your `forward` (L12, [L25](https://github.com/MPAS-Dev/compass/blob/f8af494f5c277c381d1b02320c1b9e297d01f9da/compass/ocean/tests/turbulence_closure/forward.py#L25)) and `initial_state` (L22, L34) still carry comments where resolution is expected to be string, though it is the float that is passed (as far as I understand). It may be good to propagate the update.
3) The [`run`](https://github.com/MPAS-Dev/compass/blob/f8af494f5c277c381d1b02320c1b9e297d01f9da/compass/ocean/tests/turbulence_closure/forward.py#L87-L93) in `forward` still assumes a string resolution with units. I couldn't find a call for it so maybe it's just stale code, but worth updating in case it ever get exercised.